docs/build-on-linea/linea-version/index.mdx

```diff
@@ -13,11 +13,99 @@ import TokenBridgeL2 from "!!raw-loader!/files/testnet/2023-06-13-release/TokenB
 
 Find out all you need to know about the latest Linea versions.
 
-> ### **Current network versions**
->
-> | Network | Version | Deployment Date |
-> | --- | --- | --- |
-> | Linea Goerli | [Alpha v0.2.3](#linea-alpha-v023-june-22-23) | June 22, 2023 |
+## Alpha v2 Release Notes
+
+<table>
+    <tbody>
+      <tr>
+        <th>Network</th>
+        <th>Target Release Date</th>
+      </tr>
+      <tr>
+        <td>Mainnet</td>
+        <td>February 8th, 2024</td>
+      </tr>
+      <tr>
+        <td>Testnet</td>
+        <td>January 24, 2024</td>
+      </tr>
+    </tbody>
+</table>
+
+### Summary
+
+This release focuses on reducing L1 costs, enabling lower prices for end users. 
+
+Primarily, the L1 costs are driven by the following:
+
+- Proof verification, which is essentially a fixed cost for every batch submission.
+
+- Hashing the batch submission data to produce the public input of the Plonk verifier.
+
+- Calldata costs, driven by the size of the batch submission data. This includes all the transaction data from L2, as well as messaging data (one hash per L2 to L1 message).
+
+### Features
+
+To decrease L1 costs, we implemented the following features: 
+
+- Proof aggregation:
+  - This allows us to create a set of proofs for conflated batches and generates an aggregated proof that verifies that all the  proofs are correct. Since it takes the same amount of gas to verify an aggregated proof vs. a proof for a single batch of conflated blocks, the average cost of verifying a proof aggregating N proofs is N times cheaper. 
+- Data compression:
+  - This reduces the call data cost on L1 by compressing L2 block data. (Post EIP-4844 this will be stored in blobs)
+- Switched L2 to L1 messaging anchoring from individual hashes to Merkle trees:
+  - Due to the size of the Merkle trees (32 messages) requiring only one Merkle root to be anchored, there is a 32x calldata size reduction. 
```

Comment:
```suggestion
  - Due to the size of the Merkle trees (32 messages) requiring only one Merkle root to be anchored, there is a 32x calldata size reduction for message hashes
```

Won't hurt making it clear that 32x calldata size reduction here is specifically for message hashes only